### PR TITLE
Feature/team

### PR DESCRIPTION
**Pull request by Gatchang-nyawargak**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by caren-chepngetich — DISMISSED** [review]

**Review by Trecy-Malonza — APPROVED** [review]

File: umuhinzi-react/.gitignore [hunk 1]
```diff
@@ -20,5 +20,8 @@
 
 npm-debug.log*
 yarn-debug.log*
+yarn-error.log
 yarn-error.log*
-.node_modules
\ No newline at end of file
+
+.node_modules
```
Comment (hunterachieng) [inline umuhinzi-react/.gitignore:26]: This already exists at line 4

File: umuhinzi-react/src/About/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import './index.css';
 
 const About = () => (
+    <div className='body'>
```
Comment (hunterachieng) [inline umuhinzi-react/src/About/index.js:4]: Why would you call this body?

File: umuhinzi-react/src/Team/index.css [hunk 3]
```diff
@@ -0,0 +1,218 @@
+body{
+  margin-top: 6%;
```
Comment (hunterachieng) [inline umuhinzi-react/src/Team/index.css:2]: This will affect everyone's code. Please remove it

File: umuhinzi-react/src/Team/index.css [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+  .team-member img {
+    width: 100px;
+    height: 100px;
+    gap: 30px;
+    object-fit: cover;
+    border-radius: 50%;
+    margin-bottom: 1rem;
+  }
+
+  .team-member p {
```
Comment (hunterachieng) [inline umuhinzi-react/src/Team/index.css:184]: Remove the comments

**Review by Trecy-Malonza — APPROVED** [review]

**Review by hunterachieng — APPROVED** [review]

**Review by Linetmkandoe — APPROVED** [review]

**Review by ChristineAkinyi — APPROVED** [review]

**Review by caren-chepngetich — APPROVED** [review]

**Pull request merged**